src/Generator/Generators/CodeGenerator.cs

```diff
@@ -61,14 +61,6 @@ protected CodeGenerator(BindingContext context, IEnumerable<TranslationUnit> uni
 
         public abstract void Process();
 
-        public override string Generate()
-        {
-            if (Options.IsCSharpGenerator && Options.CompileCode && !Options.GenerateDebugOutput)
-                return base.GenerateUnformatted();
```

Comment:
Please also delete the definition of GenerateUnformatted and any other methods no longer used.